agent_edit_demo.py

```
@@ -0,0 +1,286 @@
+r = cmd('list_images', {})
+for _ in r.get('results', {}).get('images', []):
+    cmd('close_image', {'image_index': 0})
+
+r = cmd('open_image', {'file_path': 'C:/localMll/cruellaOutput/navi_portrait.png'})
+print(f"  Opened: {r.get('status')}  id={r.get('results',{}).get('image_id')}")
+
+# ── STEP 1: Snapshot — original state ───────────────────────────────────────
+print()
+print("STEP 1: Snapshot BEFORE edits (original)")
+raw = snapshot(label="original")
+if raw: save_png(raw, 'C:/localMll/cruellaOutput/snap_01_original.png')
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)